.github/workflows/android.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Setup environment
       run: |
         pip install -e .
-        pip install Cython==0.29.19
+        pip install Cython
```

Comment:
We use to have issues years ago with Cython versions, but it's no longer the case, so I don't see a reason to pin it until issues are back